packages/ui/src/components/ProfileStatusSheet.tsx

```diff
@@ -57,27 +59,38 @@ export default function ProfileStatusSheet({
       snapPoints={[60]}
     >
       <ActionSheet.Content flex={1} paddingBottom={0}>
-        <YStack flex={1} marginHorizontal="$2xl">
-          <ControlledTextField
-            name="status"
-            label="Update your status"
-            control={control}
-            inputProps={{
-              placeholder: 'Hanging out...',
-              autoFocus: true,
-              returnKeyType: isValid ? 'send' : 'done',
-              onSubmitEditing: handleSave,
-              blurOnSubmit: true,
-              multiline: true,
-              maxLength: 50,
-            }}
-            rules={{
-              maxLength: {
-                value: 50,
-                message: 'Your status is limited to 50 characters',
-              },
-            }}
-          />
+        <YStack flex={1} marginHorizontal="$2xl" gap="$l">
+          <XStack gap="$m" alignItems="flex-end" width="100%">
+            <ControlledTextField
+              name="status"
+              label="Update your status"
+              control={control}
+              inputProps={{
+                placeholder: 'Hanging out...',
+                autoFocus: true,
+                returnKeyType: isValid ? 'send' : 'done',
+                onSubmitEditing: handleSave,
+                blurOnSubmit: true,
+                multiline: false,
+                maxLength: 50,
+              }}
+              flex={1}
+              rules={{
+                maxLength: {
+                  value: 50,
+                  message: 'Your status is limited to 50 characters',
+                },
+              }}
+            />
+            <Button
+              hero
+              onPress={handleSave}
+              disabled={!isValid}
+              padding={11.5}
```

Comment:
Not stoked about this, but I had to do this to make the height of the button match the height of the field.